openmsupply/application-manager-web-app
# #310B New filters by reviewer and assigner

## Branch to go along with back-end PR

### Changes
- Add new filters for `assigner` and `reviewer` filtering. 
- Problem is that we don't set the `assigner_id` yet, so can't test that. 
- To test with `reviewer` I was using:
Go to see the list of `ReviewTest` applications
- Test 1: Filter by "testReviewer1" with URL `http://localhost:3000/applications?type=ReviewTest&user-role=consolidator&reviewer=testReviewer1` who is assigned to all applications, so nothing changes.
- Test 2: Filter by "testReviewer2" with URL`http://localhost:3000/applications?type=ReviewTest&user-role=consolidator&reviewer=testReviewer2` now only 3 applications will show because this user isn't assigned to one of the applications
